src/nvim/tui/tui.c

```diff
@@ -1021,10 +1053,11 @@ static void fix_terminfo(TUIData *data)
 end:
   // Fill some empty slots with common terminal strings
   if (data->term == kTermiTerm) {
-    data->unibi_ext.set_cursor_color = (int)unibi_add_ext_str(
+    data->unibi_ext.set_cursor_bg_color = (int)unibi_add_ext_str(
         ut, NULL, "\033]Pl%p1%06x\033\\");
   } else {
-    data->unibi_ext.set_cursor_color = (int)unibi_add_ext_str(
+    // new form rgb:X/X/X does not seem to work so well
```

Comment:
It's up to date but might be more relevant in the git log. rgb:X/X/X is supposed to work (see xparsecolor) but there are different forms such as rgb:XXX/XXX/XXX, rgb:XX/XX/XX and not all had an effect in termite. I would have to ask on vte tracker.